QuestPatcher.QMod/QModManifest.cs

```diff
@@ -137,10 +137,11 @@ public string VersionString
         {
             "0.1.0",
             "0.1.1",
-            "0.1.2"
+            "0.1.2",
+            "1.0.0"
         }.ToHashSet();
 
-        private const string LatestSchemaVersion = "0.1.2";
+        private const string LatestSchemaVersion = "1.0.0";
 
         private static readonly JsonSerializerOptions SerializerOptions = new JsonSerializerOptions()
         {
```

Comment:
In this file, consider resolving the potential null references to the `QModManifest` type before merging this featureset.